go.mod

```diff
@@ -46,9 +47,34 @@ require (
 	gotest.tools v2.2.0+incompatible
 	k8s.io/api v0.20.2
 	k8s.io/apimachinery v0.20.2
-	k8s.io/client-go v0.20.2
+	k8s.io/client-go v10.0.0+incompatible
 	k8s.io/utils v0.0.0-20210111153108-fddb29f9d009
 	sigs.k8s.io/controller-runtime v0.8.2
 )
 
-replace github.com/aws/amazon-sagemaker-operator-for-k8s => github.com/aws/amazon-sagemaker-operator-for-k8s v1.0.1-0.20210303003444-0fb33b1fd49d
+replace (
+	github.com/aws/amazon-sagemaker-operator-for-k8s => github.com/aws/amazon-sagemaker-operator-for-k8s v1.0.1-0.20210303003444-0fb33b1fd49d
+	k8s.io/api => k8s.io/api v0.20.2
```

Comment:
this looks bad!!!!!!!!!!!!

Reply:
yeah, I am aware of the issue.

Reply:
@kumare3  https://github.com/kubernetes/kubernetes/issues/79384

Reply:
i dont think we need kube api

Reply:
Just found out there is a `/v2` version of the package that Yuvraj can use...
https://github.com/kubeflow/mpi-operator/tree/master/v2

`go get github.com/kubeflow/mpi-operator/v2`

It's currently on 0.19... we can probably fork it and propose a PR to upgrade to 0.20